controller.go

```diff
@@ -262,6 +270,38 @@ func (c Controller) htmlHandler(w http.ResponseWriter, r *http.Request) {
 
 }
 
+func formatServiceName(name string) string {
+	loc := serverInstanceRegex.FindStringIndex(name)
+	if loc == nil {
+		return name
+	}
+	nameAsRunes := []rune(name)
+	nameAsRunes[loc[0]] = serviceInstanceDelimiter
```

Comment:
Was just thinking if we ever gonna have cases when the service name would contain characters which are represented as more than one bytes, but I don't think so, so no problem here :)